app/assets/stylesheets/iu-branding.scss

```diff
@@ -0,0 +1,124 @@
+@import "iu_variables";
+
+// links (e.g. in the menu bar)
+a {
+  color: $iu-crimson;
+}
+a:hover {
+  color: $iu-mahogany;
+}
+.page-link {
+  color: $iu-crimson;
+}
+.page-link:hover {
+  color: $iu-mahogany;
+}
+.page-item.active .page-link {
+  background-color: $iu-crimson;
+}
+
+// button colors
+.btn-primary {
+  background-color: $iu-crimson;
+  border-color: $iu-mahogany;
+}
+.btn-primary:hover, .btn-primary:active {
+  background-color: $iu-mahogany;
+}
+.btn-secondary{
+  background-color: $iu-mahogany;
+}
+.btn-outline-secondary {
+  border-color: $iu-mahogany;
+}
+
+// The "X" button to remove a facet from your search results
+.btn-danger:hover, .applied-filter .remove:hover  {
+  background-color: $iu-cream;
+}
+
+// active colors for toggable buttons on search results
+.btn-outline-secondary:not(:disabled):not(.disabled):active, .btn-outline-secondary:not(:disabled):not(.disabled).active,
```

Comment:
for future reference, when you have multiple selectors that are separated by commas, you should put them on separate lines for readability.